acceptance/acceptance_suite_test.go

```diff
@@ -426,6 +427,24 @@ func Epinio(command string, dir string) (string, error) {
 	return RunProc(cmd, commandDir, false)
 }
 
+// GetKCommand returns the KCommand without running it
+func GetKCommand(command string, dir string) (*kexec.KCommand, error) {
```

Comment:
Could this be used to simplify the preceding `Epinio` function ?

Reply:
Fixed here: https://github.com/epinio/epinio/commit/414aad727d43d33268d80c057487aed2b21aeacc